packages/compartment-mapper/src/types.d.ts

```diff
@@ -0,0 +1,11 @@
+// ESlint does not understand export *
+/* eslint-disable import/export */
+
+export * from './types/external.ts';
+export * from './types/internal.ts';
+export * from './types/policy-schema.ts';
+export * from './types/compartment-map-schema.ts';
+export * from './types/policy.ts';
+export * from './types/powers.ts';
+export * from './types/node-powers.ts';
+export * from './types/typescript.ts';
```

Comment:
These should all end in `.js` afaik?

Reply:
It works either way and I don’t have a strong preference. I originally wrote these with `.js` and @turadg recommended `.ts`. I don’t have a principled position to stand on either way.

Reply:
I’ve changed all of these to `.js`.

Reply:
I still see the specifiers here using `.ts`

Reply:
This conversation is tied to an outdated delta.